.env.example

```diff
@@ -7,11 +7,16 @@ METRIQ_CLIENT_API_KEY="<METRIQ_CLIENT_API_KEY>"
 
 # IBM
 # Obtained at: https://quantum.ibm.com
-IBM_QISKIT_API_KEY="<IBM_QISKIT_API_KEY>"
+QISKIT_IBM_TOKEN="<QISKIT_IBM_TOKEN>"
```

Comment:
Do we want to abide by the same pattern of `<PROVIDER_API_KEY>`? So, IOW, `IBM_API_KEY = ...`?

Reply:
The way the framework works now, the name of these variables need to match their names in qBraid, e.g., https://github.com/qBraid/qBraid/blob/7bd29d90c2938713364766dc30545456b5126a5f/.env.example#L5. We can change this in the future by creating "wrapper" variables, but for now, let's just stick to their names.